tests/dataconverter/test_helpers.py

```diff
@@ -269,12 +295,13 @@ def test_validate_data_dict(data_dict, error_message, template, nxdl_root, reque
                                     "no-child-provided-optional-parent",
                                     "int-instead-of-chars",
                                     "link-dict-instead-of-bool",
-                                    "allow-required-and-empty-group"):
+                                    "allow-required-and-empty-group",
+                                    "opt-group-completely-removed"):
         helpers.validate_data_dict(template, data_dict, nxdl_root)
     else:
         with pytest.raises(Exception) as execinfo:
             helpers.validate_data_dict(template, data_dict, nxdl_root)
-
+        print(execinfo.value)
```

Comment:
Is this planned or just a residual from debugging?

```suggestion
```

Reply:
Ah. Residual ofc...